src/DssSpell.sol

```diff
@@ -17,53 +17,38 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral
-pragma experimental ABIEncoderV2;
+// pragma experimental ABIEncoderV2;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import "dss-interfaces/dss/FlapAbstract.sol";
 
 import { DssSpellCollateralOnboardingAction } from "./DssSpellCollateralOnboarding.sol";
 
+interface Gem6Like {
+    function implementation() external returns (address);
+}
+
+interface GemJoin6Like {
+    function setImplementation(
+        address implementation,
+        uint256 permitted
+    ) external;
+}
+
 contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/19e338132d799741f42f07a130efb9692654897f/governance/votes/Executive%20Vote%20-%20March%2011%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO/governance/votes/Executive%20Vote%20-%20March%2025%2C%202022.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-03-11 MakerDAO Executive Spell | Hash: 0x4eaeba62293ee8c19ccb1dfc7d9e97b5764b7e62fefebc1049f2e8f984d0836f";
-
-    // Immunefi Payouts
-    address constant WALLET1 = 0x3C32F2ca11D92a7093d1F237161C1fB692F6a8eA;
-    address constant WALLET2 = 0x2BC5fFc5De1a83a9e4cDDfA138bAEd516D70414b;
-
-    // Flap
-    address constant MCD_FLAP = 0xa4f79bC4a5612bdDA35904FDF55Fc4Cb53D1BFf6;
-
-    // Math
-    uint256 constant RAD = 10**45;
+        "2022-03-25 MakerDAO Executive Spell | Hash: TODO";
```

Comment:
Need to turn off office hours with the function override.

Reply:
done